### PR TITLE
Second day on Git

### PR DESCRIPTION
**Pull request by BrianNguyen420**
Here is the update to mentor Khoa about how I practice on GIT.

### CODE REVIEW

**Comment by rubikvn2100** [comment]
Good Job on solving the first problem.

File: addition.test.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import unittest
+from addition import addition
+
+class TestSumFuction(unittest.TestCase):
```
Comment (rubikvn2100) [inline addition.test.py]: You should change "Sum" to "Addition".

File: multiply.test.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import unittest
+from multiply import multiply
+
+class TestMulFuction(unittest.TestCase):
```
Comment (rubikvn2100) [inline multiply.test.py]: You should change "Mul" to "Multiply"

**Review by rubikvn2100 — CHANGES_REQUESTED** [review]

File: addition.test.py (hunk 1 above)
Comment (BrianNguyen420) [reply addition.test.py]: I understand your request.

File: multiply.test.py (hunk 2 above)
Comment (BrianNguyen420) [reply multiply.test.py]: I acknowledge your request.

**Comment by rubikvn2100** [comment]
Great, keep going main.

**Pull request merged**